integration/ct_integration_test.sh

```diff
@@ -3,6 +3,10 @@ set -e
 INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
 . "${INTEGRATION_DIR}"/common.sh
 
+yes | "${SCRIPTS_DIR}"/resetdb.sh
```

Comment:
I'd previously avoided using `resetdb.sh` as it blows everything away (my original intent for that script was coping with schema changes).  However, it's the `test` database only so it's probably fair game to wipe it.

"I say we take off and nuke the entire site from orbit. It's the only way to be sure."